### PR TITLE
add note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/Viva/issues/754

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Starshine89 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by Starshine89 — COMMENTED** [review]
Looks good!

**Review by Starshine89 — APPROVED** [review]

**Review by Starshine89 — CHANGES_REQUESTED** [review]
Please change this to "Note" to be consistent with other Viva Engage articles. Thanks!

File: Viva/engage/campaigns.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Campaigns are an interactive way to drive employee engagement. They provide powe
 Creating and viewing the campaign landing page and analytics from the campaign management dashboard is only available to licensed admins. Unlicensed network, global, and engage admins can only access the campaign management dashboard and delete campaigns that were created by licensed admins or corporate communicators.
 
 ## Set up a campaign
+
+> [!IMPORTANT]
```
Comment (JohanFreelancer9) [inline Viva/engage/campaigns.md:31]:
```suggestion
> [!Note]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
As per @Starshine89's request.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Starshine89 — APPROVED** [review]

**Pull request merged**